collectors/iface/collector.go

```diff
@@ -80,16 +81,23 @@ func (Collector) Collect(ch chan<- prometheus.Metric) {
 		if !ok {
 			continue
 		}
-		if i.Type == "" {
-			// empty string is a synonym for "system"
-			i.Type = "system"
-		}
```

Comment:
why did you remove this block?

Reply:
I somehow managed to delete commit 5fcad7b which contained this change in my branch.